pom.xml

```diff
@@ -215,6 +215,11 @@
                 <artifactId>flume-ng-core</artifactId>
                 <version>${flume.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.tomcat.embed</groupId>
+                <artifactId>tomcat-embed-core</artifactId>
+                <version>8.5.63</version>
```

Comment:
please add the version definition to the root `pom`, it will help keep the same version for other using scenes.